src/iuse.cpp

```diff
@@ -7808,3 +7808,40 @@ int iuse::washclothes( player *p, item *it, bool, const tripoint & )
 
     return 0;
 }
+int iuse::break_stick( player *p, item *it, bool, const tripoint &pos )
+{
+    p->moves -= 200;
+    p->mod_stat( "stamina", -50.0f * p->stamina / p->get_stamina_max() );
+
+    if( p->get_str() < 5 ) {
+        p->add_msg_if_player( _( "You are too weak to even try." ) );
+        return 0;
+    } else if( p->get_str() <= rng( 5, 11 ) ) {
+        p->add_msg_if_player(
+            _( "You use all your strength, but the stick won't break.  Perhaps try again?" ) );
+        return 0;
+    }
+
+    if( !p->has_item( *it ) ) {
+        g->m.i_rem( pos, it );
+    } else {
+        p->i_rem( it );
+    }
```

Comment:
The stick won't be deleted if it was stored in a vehicle. Perhaps you should use `consume_items` function for this, like I did in (https://github.com/CleverRaven/Cataclysm-DDA/pull/24081/commits/68d2a0ded05e8535dd0f3a532b603db4c9e371d6) commit.